state/elasticache-redis.yaml

```diff
@@ -72,12 +72,16 @@ Parameters:
   EngineVersion:
     Description: 'Redis version'
     Type: String
-    Default: '5.0.0'
+    Default: '6.x'
```

Comment:
Wh changing the default (breaking change)?

Reply:
Good point. Will keep the old default.